docs/sources/github/README.md

```diff
@@ -6,3 +6,37 @@ There are several connectors available for GitHub:
 - [GitHub non enterprise](github-non-enterprise/README.md) - for non-Enterprise GitHub Cloud (Free/Pro/Teams) organization hosted in github.com.
 - [GitHub Enterprise Cloud](enterprise-server/README.md) - GitHub Enterprise instances hosted by github.com on behalf of your organization.
 - [GitHub Copilot](copilot/README.md) - Copilot data from GitHub Enterprise instances hosted by github.com on behalf of your organization.
+
+
+## Troubleshooting
+
+There are THREE components to auth:
+  - `PRIVATE_KEY` --> filled by you directly in your host's secret/parameter store,
+  - `CLIENT_ID` --> filled by you directly in your host's secret/parameter store,
+  - `github_copilot_installation_id` --> fill in your `terraform.tfvars` file with the installation id of your GitHub App, which is used to generate the `REFRESH_URL` env var on the lambda/cloud function.
+
+Common pitfalls:
+  - creating a OAuth App instead of a GitHub App ; these are NOT the same thing.
```

Comment:
I'd put more detail here:
```suggestion
  - creating a OAuth App instead of a GitHub App ; these are NOT the same thing; we use GitHub app as it acts independently of the user. Check this [link](https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/differences-between-github-apps-and-oauth-apps) with more information about its differences. 
```